examples/service/ocean/providers/azure_np/create/main.go

```diff
@@ -0,0 +1,107 @@
+package main
+
+import (
+	"context"
+	"github.com/spotinst/spotinst-sdk-go/service/ocean/providers/azure_np"
+	"log"
+
+	"github.com/spotinst/spotinst-sdk-go/service/ocean"
+	"github.com/spotinst/spotinst-sdk-go/spotinst"
+	"github.com/spotinst/spotinst-sdk-go/spotinst/session"
+	"github.com/spotinst/spotinst-sdk-go/spotinst/util/stringutil"
+)
+
+func main() {
+	// All clients require a Session. The Session provides the client with
+	// shared configuration such as account and credentials.
+	// A Session should be shared where possible to take advantage of
+	// configuration and credential caching. See the session package for
+	// more information.
+	sess := session.New()
+
+	// Create a new instance of the service's client with a Session.
+	// Optional spotinst.Config values can also be provided as variadic
+	// arguments to the New function. This option allows you to provide
+	// service specific configuration.
+	svc := ocean.New(sess)
+
+	// Create a new context.
+	ctx := context.Background()
+
+	// Create a new cluster.
+	out, err := svc.CloudProviderAzureNP().CreateCluster(ctx, &azure_np.CreateClusterInput{
+		Cluster: &azure_np.Cluster{
+			Name:                spotinst.String("foo"),
+			ControllerClusterID: spotinst.String("foo"),
+			AKS: &azure_np.AKS{
+				ClusterName:                     spotinst.String("foo"),
+				ResourceGroupName:               spotinst.String("foo"),
+				Region:                          spotinst.String("foo"),
+				InfrastructureResourceGroupName: spotinst.String("foo"),
+			},
+			AutoScaler: &azure_np.AutoScaler{
+				IsEnabled: spotinst.Bool(true),
+				ResourceLimits: &azure_np.ResourceLimits{
+					MaxVCPU:      spotinst.Int(120),
+					MaxMemoryGib: spotinst.Int(120),
+				},
+				Down: &azure_np.Down{
+					MaxScaleDownPercentage: spotinst.Float64(30),
```

Comment:
Lets give float value. Also test in terraform witrh float value.

Reply:
added float value